kuma/static/styles/components/wiki/document-head.scss

```diff
@@ -46,3 +47,5 @@
         @include large-overflow();
     }
 }
+
+*/
```

Comment:
Nit: I generally prefer to delete things as apposed to comment them out seeing that we are using a source control system. There is always the chance that one forgets to remove it should you never enable it again, which can lead to code bloat. I guess SASS will help here but, just a thought.